js/app.js

```diff
@@ -217,8 +217,9 @@ function buildContentSnippet(blogSummary, data, key) {
     const objSnippet = blogSummary.querySelector("[data-codename='" + key + "']");
     if (objSnippet !== null) {
         const elementSnippet = document.createElement(snippetType.tag);
-        if (snippetType.format === "text") elementSnippet.textContent = firstOrDefaultValue(data[key], snippetType.property) || "";
-        else elementSnippet.innerHTML = firstOrDefaultValue(data[key], snippetType.property) || "";
+        const snippetContent = firstOrDefaultValue(data[key], snippetType.property) || "";
+        if (snippetType.format === "text") elementSnippet.textContent = snippetContent;
+        else elementSnippet.innerHTML = snippetContent;
```

Comment:
Some people prefer a ternary expression here but I think it's down to personal preference. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator